### PR TITLE
Auth flow bugs

### PR DESCRIPTION
**Pull request by theibrahim**

### CODE REVIEW

**Comment by theibrahim** [comment]
Conflicts fixed

File: components/SignIn.js [hunk 1]
```diff
@@ -18,35 +18,47 @@ const SignIn = ({ props }) => {
   const [email, setEmail] = useState("");
   const [loading, setLoading] = useState(false);
   const [isLoginEmailSent, setIsLoginEmailSent] = useState(false);
-  const [signInError, setSignInError] = useState("");
 
-  const handleSubmit = (e) => {
+  // const [loginEmailSentToEmail, setLoginEmailSentToEmail] = useState("");
```
Comment (pauloouriques) [inline components/SignIn.js]: remove it

File: components/SignIn.js [hunk 2]
```diff
@@ -18,35 +18,47 @@ const SignIn = ({ props }) => {
...
+      return false;
+    }
+
+
+    try {
+      await signIn("email", { email, redirect: false });
+
+      setLoading(false);
+      setIsLoginEmailSent(true);
+      // setLoginEmailSentToEmail(email);
```
Comment (pauloouriques) [inline components/SignIn.js]: remove

File: components/SignIn.js [hunk 3]
```diff
@@ -62,6 +74,19 @@ const SignIn = ({ props }) => {
     };
   }, [email]);
 
+  // const sendAgainMagicLink = async () => {
```
Comment (pauloouriques) [inline components/SignIn.js]: remove it

File: components/SignIn.js [hunk 4]
```diff
@@ -121,21 +146,61 @@ const SignIn = ({ props }) => {
...
+                        Magic Sign In link sent! Check your{" "}
+                        <a
+                          className="underline text-primary"
+                          href="https://mail.google.com/"
+                          target="_blank"
+                        >
+                          email
+                        </a>
+                      </h6>
+                      {/* <p className="text-gray-700 ">
```
Comment (pauloouriques) [inline components/SignIn.js]: remove it

File: pages/business/index.js [hunk 5]
```diff
@@ -27,11 +27,11 @@ export default function Home() {
   const [activeDeleteIndex, setActiveDeleteIndex] = useState(null);
   const [tagEditModal, setTagEditModal] = useState(false);
   const [activeTagIndex, setActiveTagIndex] = useState(0);
-  const [notMatchesPage, setNotMatchesPage] = useState(1);
+  // const [currentPage, setCurrentPage] = useState(1);
```
Comment (pauloouriques) [inline pages/business/index.js]: remove it

File: pages/business/index.js [hunk 6]
```diff
@@ -57,7 +73,9 @@ export default function Home() {
         if (res.status === 200) {
           const data = await res.json();
           setListings(data.results);
+          console.log(data.pagination);
```
Comment (pauloouriques) [inline pages/business/index.js]: clean up all console.logs that are only for debug/testing

**Review by pauloouriques — CHANGES_REQUESTED** [review]
clean up

**Comment by theibrahim** [comment]
I am on it

**Comment by theibrahim** [comment]
Thanks for feedback

**Comment by theibrahim** [comment]
removed all unnecessary code from all places
also fixed conflicts

**Pull request merged**